### ISSUE TITLE
How to update to the latest ESR version (or to a specific version)?

### ISSUE THREAD

**Issue by R8s6** [mattermost/mattermost-docker#532]
Hi,

TL;DR: Is there a way to upgrade to the latest ESR version?

I installed following the default guide, which is probably tracking the `master` branch.

At the time of install, it happened to be v5.31.0.

If I'd only wish to track the [ESR releases](https://docs.mattermost.com/administration/changelog.html#): v5.31.3 as of today, and to the next ESR version in the future, is there a way to achieve this?

Since we don't seem to have an "ESR" branch, as a workaround, should I back up and start over by git cloning to the `release-5.31` branch, and, in the near future, upgrade normally (with `git pull`); and when next ESR is out, start over again with that branch and repeat?

Thanks!

**Comment by elandorr** [comment]
Since the mattermost team doesn't care about non-enterprise users (as we also see with the insane lack of retention policy/maintenance/permissions in the team edition) I'll tell you what I did:

Basically what you did already:

1. Look up version here: https://docs.mattermost.com/administration/extended-support-release.html
2. `git clone -b release-5.31` in a new folder
3. I keep the volumes as local folders and not the messy docker volumes. Just restore the data, copy the folder.
4. If everything works, delete the old folder.

This is fast, clean and spares us from their issues. The issue referenced above is almost a year old by now. Goes to show how much they care.
Until there is a better alternative I'll still stick with MM. 'best of the worst' is not really nice though.
If you make tons of money with an already enterprise-only model (since lack of retention policy/any administration or permission features whatsoever basically excludes everyone but personal ultra-small scale setups by default) you really could at least care enough to take care of maintenance issues.

(I know some user already went through the trouble of creating a little fix-it-your-self guide in a comment ages ago, but I do not want to mess with a database until the developers decided what is going to happen and how to move forward. But the ESR is nice for stability anyway!)
